inst/stan/functions/expected_obs.stan

```diff
@@ -58,37 +65,51 @@
  * rep_lh <- rep(0, 30)
  * rep_lh[c(6, 12, 16)] = 2
  * rep_lh[c(2, 20)] = -2
- * round(exp(expected_obs(tar_obs, equal_lh + rep_lh, 0)), 3)
+ * round(exp(expected_obs(tar_obs, equal_lh + rep_lh, 0, 0, [0])), 3)
  * # 0.033 0.005 0.034 0.034 0.034 0.202 0.027 0.027 0.027 0.027 0.027 0.151
  * # 0.021 0.021 0.021 0.106 0.014 0.014 0.014 0.002 0.016 0.016 0.016 0.016
  * # 0.016 0.016 0.016 0.016 0.016 0.016
  *
  * Example combining both date of reference and date of report effects
- * eobs <- exp(expected_obs(tar_obs, ref_lh + rep_lh, 0))
+ * eobs <- exp(expected_obs(tar_obs, ref_lh + rep_lh, 0, 0, [0]))
  * round(sum(eobs - 1e-4), 5) == 1
  * round(eobs, 3)
  * # 0.273 0.023 0.112 0.082 0.065 0.221 0.025 0.021 0.019 0.016 0.014 0.058
  * # 0.007 0.007 0.006 0.024 0.003 0.003 0.003 0.000 0.003 0.002 0.002 0.002
  * # 0.002 0.002 0.002 0.002 0.002 0.002
+ *
+ * Example with aggregation of probabilities where aggregation occurs on every fifth day
+ * agg_probs <- matrix(c(rep(0, times = 30 * 4),
+ *                       rep(c(rep(1, times = 5),
+ *                             rep(0, times = 6 * 5 + 30 * 4)),
+ *                           times = 5),
+ *                       rep(1, times = 5)), ncol = 30, byrow = TRUE)
+ * eobs <- exp(expected_obs(tar_obs, date_p + rep(0, 30), 1, 1, agg_probs))
```

Comment:
when I ran this locally I found having `exp_obs |> exp() |> plot()` this to be helpful. This looks like it produces the expected output and that output makes sense to me.